src/email/logs.controller.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('Email log Controller', function() {
+    var scope, controller;
+
+    beforeEach(module('motech-email'));
+    beforeEach(inject(function($rootScope, $controller){
+        scope = $rootScope.$new();
+        controller = $controller('EmailLogsController', {$scope: scope});
+    }));
+
+    it('should return good data from scope.checkboxes', function() {
+        expect(scope.checkboxes[0]).toEqual({label: "email.logging.error", value: "ERROR"});
+        expect(scope.checkboxes[1]).toEqual({label: "email.logging.sent", value: "SENT"});
+        expect(scope.checkboxes[2]).toEqual({label: "email.logging.received", value: "RECEIVED"});
+        expect(scope.checkboxes[0].value).toEqual("ERROR");
```

Comment:
Is it necessary? If we check if a `scope.checkbox[0]` has fields `label` and `value` above, there is no need to check the value field separately again.